lib/cc/analyzer/container.rb

```diff
@@ -178,9 +176,11 @@ def kill_wait_thread
 
       def reap_running_container(message)
         Analyzer.logger.warn("killing container name=#{@name} message=#{message.inspect}")
-        POSIX::Spawn::Child.new("docker", "kill", @name, timeout: 2.minutes)
-        POSIX::Spawn::Child.new("docker", "wait", @name, timeout: 2.minutes)
-      rescue POSIX::Spawn::TimeoutExceeded
+        Timeout.timeout(2.minutes.to_i) do
+          Kernel.system("docker kill #{@name}")
+          Kernel.system("docker wait #{@name}")
```

Comment:
I can imagine that originally this was because the `docker kill` command wasn't a blocking one? so they had to `wait` for the container to exit.

As today, those two line together don't make sense, as the `kill` command is blocking.

Again, it's a piece of code that I don't want to change in this already long PR